docs/release-notes/release-notes-0.19.0.md

```
@@ -25,6 +25,10 @@
 
 # New Features
 ## Functional Enhancements
+
+* Add route blinding receive functionality to for [hodl 
```

Markdownlint

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)